ansible_ai_connect/healthcheck/views.py

```diff
@@ -55,12 +55,14 @@ def common_data():
 class HealthCheckCustomView(MainView):
     _plugin_name_map = {
         "DatabaseBackend": "db",
-        "ModelServerHealthCheck": "model-server",
         "AWSSecretManagerHealthCheck": "secret-manager",
-        "WCAHealthCheck": "wca",
-        "WCAOnPremHealthCheck": "wca-onprem",
         "AuthorizationHealthCheck": "authorization",
-        "ChatbotServiceHealthCheck": "chatbot-service",
+        "ModelPipelineCompletions": "completions-service",
```

Comment:
I'd like to move the alias to the pipeline itself and make this block more _dynamic_.